lib/net.c

```diff
@@ -1108,6 +1123,14 @@ start_iface(struct gatekeeper_if *iface)
 		goto stop_partial;
 
 out:
```

Comment:
It's not related to the patch itself, but the label `out` here is being misused because it doesn't stack to error handling labels. You can combine this and the other comment about `start_iface()` in another patch to keep this patch coherent.

My suggestion is:
1. to invert `if (iface->num_ports <= 1 && iface->bonding_mode != BONDING_MODE_8023AD)`: `if (iface->num_ports > 1 || iface->bonding_mode == BONDING_MODE_8023AD)`;
2. Move the code before the label `out` into the inverted `if` statement;
3. drop the label `out:`;
4. Rewrite the comment `/* If there's no bonded port, we're done. */` and put it at the beginning of the body of the inverted `if` statement: `/* Bonding port(s). */`